addons/api/.eslintrc.js

```diff
@@ -9,14 +9,12 @@ module.exports = {
   root: true,
   parser: '@babel/eslint-parser',
   parserOptions: {
-    ecmaVersion: 'latest',
+    ecmaVersion: 2018,
```

Comment:
Why the change back to `2018` and legacy?

Reply:
@ZedLi  So this one I couldn't find much of a explanation for it, but it seems to be part of the `@babel/plugin-proposal-decorators`.   Here is the failing [job](https://github.com/hashicorp/boundary-ui/actions/runs/6618893361/job/17978429268).   My guess is its because the `parserOptions` are not the same across for every addon and admin-ui?  Any thoughts here would be great

Reply:
Does it happen locally?

Reply:
@ZedLi no

Reply:
It would be good to investigate why the CI fails then and what's the difference between local and the job

Reply:
@ZedLi looks like the previous failed CI jobs are clean now.   Still not 100% what the issue was, but possibly the mismatch versions of `@babel/eslint-parser`.